modules/sdk-core/src/bitgo/utils/tss/baseTypes.ts

```diff
@@ -195,6 +195,7 @@ export interface PrebuildTransactionWithIntentOptions extends IntentOptionsBase
   custodianTransactionId?: string;
   receiveAddress?: string;
   unspents?: string[];
+  senderAddress?: string;
```

Comment:
please add jsdoc - under which circumstances is it set and what's the value

Reply:
addressed